sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 #### Bugs Fixed
 
+* Enabled read failover to preferred locations in the case of single-write/multi-read region enabled account. See [Bug: Cosmos DB Client gets stuck in timeout retry loop](https://github.com/Azure/azure-sdk-for-java/issues/31260#issue-1396454421)
```

Comment:
```suggestion
* Enabled read failover to preferred locations in the case of single-write/multi-read region enabled account in Gateway mode. See [Bug: Cosmos DB Client gets stuck in timeout retry loop](https://github.com/Azure/azure-sdk-for-java/issues/31260#issue-1396454421)
```

Reply:
`Enabled failover to preferred locations in the case of single-write/multi-read region enabled account for read in Gateway mode and for metadata requests in Direct mode. See [Bug: Cosmos DB Client gets stuck in timeout retry loop](https://github.com/Azure/azure-sdk-for-java/issues/31260#issue-1396454421)`